pkg/storage/postgresql/issues.go

```diff
@@ -279,6 +289,8 @@ func (db DB) IssuesSummary(filter storage.Filter) ([]model.IssueSummary, storage
 					{{ end }}
 				{{ end }}
 				{{ if and (eq .Status "OPEN") (or .MinDate .MaxDate) }}
+					/* Ignore findings marked as false positive */
+					AND f.status != 'FALSE_POSITIVE'
 					{{ if .MinDate }}
```

Comment:
There is a mismatch between the ignored FALSE POSITIVE conditions defined in the size query vs the "results query".

Reply:
You are right! I fixed it now. I will include a e2e test for it in the afternoon